docs/en/Plugin.md

```diff
@@ -166,6 +166,51 @@ dayjs.extend(isBetween)
 dayjs('2010-10-20').isBetween('2010-10-19', dayjs('2010-10-25')); // true
 ```
 
+### TimeZone
+ - TimeZone extends `dayjs()` and `dayjs().format` APIs to support the most important usage scenatrios - parsing from a specific time zone and formatting in other time zone.
+
+```javascript
+import timeZone from 'dayjs/plugin/timeZone'
+
+dayjs.extend(timeZone)
+
+const timeZone = 'Europe/Berlin' // Canonical time zone name
+
+// String without a time zone from the date picker
+const enteredDate = '2018-09-02 23:41:22'
+const storedDate = dayjs(enteredDate, { timeZone }).toISOString()
+// Will contain "2018-09-02T21:41:22Z"
+
+// String in UTC or anything that Day.js accepts
+const storedDate = '2018-09-02T21:41:22Z'
+const userFormat = 'D.M.YYYY H:mm:ss [GMT]Z (z)' // Standard Day.js format
+const displayedDate = dayjs(storedDate).format(userFormat, { timeZone })
+// Will contain "2.9.2018 23:41:22 GMT+02:00 (CEST)"
+```
+
+List of added formats:
+
+| Format | Output | Description             |
+| ------ | ------ | ----------------------- |
+| `z`    | CEST   | Time zone abbreviation  |
```

Comment:
@iamkun any chance to implement this as part of the timezone plugin?

Reply:
@akoskm  I have the same need and draft a PR here https://github.com/iamkun/dayjs/pull/1069